fork-off/src/main.rs

```diff
@@ -0,0 +1,118 @@
+use clap::Parser;
+use common::{prefix_as_hex, read_file};
+use serde_json::Value;
+use std::fs::{self, File};
+use std::io::ErrorKind;
+use std::io::Write;
+use substrate_api_client::extrinsic::log::info;
+
+#[derive(Debug, Parser)]
+#[clap(version = "1.0")]
+pub struct Config {
+    /// URL address of the node RPC endpoint for the chain you are forking
+    #[clap(long, default_value = "http://127.0.0.1:9933")]
+    pub http_rpc_endpoint: String,
+
+    /// path to write the initial chainspec of the fork
+    /// as generated with the `bootstrap-chain` command
+    #[clap(long, default_value = "../docker/data/chainspec.json")]
+    pub fork_spec_path: String,
+
+    /// where to write the forked genesis chainspec
+    #[clap(long, default_value = "../docker/data/chainspec.fork.json")]
+    pub write_to_path: String,
+
+    /// which modules to set in forked spec
+    #[clap(long, default_value = "Aura, Aleph, Treasury, Vesting")]
```

Comment:
Creating these forks will always be hands-on I suppose - so assuming person knows what he/she is doing and can always override these defaults

Reply:
If this is our assumption, I would leave this with a completely minimal set then, perhaps just `Aura` and `Aleph`. Otherwise we are incentivizing people to use the default values, while they certainly shouldn't.

Reply:
:+1: